message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1556,5 +1556,33 @@
       <field name="AOA" type="float" units="deg">Angle of Attack (degrees)</field>
       <field name="SSA" type="float" units="deg">Side Slip Angle (degrees)</field>
     </message>
+    <message id="11030" name="ESC_TELEMETRY_1_TO_4">
+      <description>ESC Telemetry Data for ESCs 1 to 4, matching data sent by BLHeli ESCs</description>
+      <field type="uint8_t[4]" name="temperature">Temperature (C)</field>
```

Comment:
How can this be a uint8_t? Negative temperatures are quite reasonable starting points.

Reply:
we're really just looking for overheat

Reply:
There should be notes that this is expected to clamp then right? And given that temp sensors are optional we need an invalid data signal anyways. (I think we really need one for every field really, or a status bitmask per ESC that tells you what fields are valid)